### ISSUE TITLE
Typings support

### ISSUE THREAD

**Issue by jaredmcateer** [shouldjs/sinon#16]
Currently if you're using typing for shouldjs using this extension breaks

`service.set_active.should.have.been.calledOnce();`
`[ts] Property 'calledOnce' does not exist on type 'Assertion'. [2339]`

**Comment by btd** [comment]
Yep, this module lack typings. Sorry I will not be able to fix any time soon. If you wanna create PR i will check it.
